job/server/src/main/java/alluxio/master/job/metrics/DistributedCmdMetrics.java

```diff
@@ -189,19 +193,22 @@ public static void incrementForCancelStatus(String jobType) {
    */
   public static void incrementForCompleteStatusWithRetry(
           JobConfig config, FileSystem fileSystem, RetryPolicy retryPolicy) {
-    String jobType = config.getName();
-    String filePath = getFilePathForNonBatchConfig(config);
-    incrementOperationCount(jobType, 1);
-    incrementFileCount(jobType, 1);
+    if (config instanceof LoadConfig || config instanceof MigrateConfig
+            || config instanceof PersistConfig) {
+      String jobType = config.getName();
+      String filePath = getFilePathForNonBatchConfig(config);
```

Comment:
this function already does the equivalent instanceof check, returning a null string if config is not one of the 3 known config types. rather than having 2 places to update if there were to be a 4th config type to increment for, can we reuse this to determine if this is a config type with countable metrics?

Reply:
yeah, i forgot to remove/modify the other place